<ui version="4.0" >
 <class>UserDialog</class>
 <widget class="QDialog" name="UserDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>194</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Repository user</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QGridLayout" name="gridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label_login" >
       <property name="text" >
        <string>Login:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" >
      <widget class="QLineEdit" name="lineEdit_login" />
     </item>
     <item row="1" column="0" colspan="2" >
      <widget class="QLabel" name="label_password" >
       <property name="text" >
        <string>Password:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" >
      <widget class="QLineEdit" name="lineEdit_password" >
       <property name="echoMode" >
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2" >
      <widget class="QLabel" name="label_group" >
       <property name="locale" >
        <locale country="UnitedStates" language="English" />
       </property>
       <property name="text" >
        <string>Group:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2" >
      <widget class="QComboBox" name="comboBox_group" >
       <item>
        <property name="text" >
         <string>USER</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>ADMIN</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_password_repeat" >
       <property name="text" >
        <string>Repeat password:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" >
      <widget class="QLineEdit" name="lineEdit_password_repeat" >
       <property name="echoMode" >
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
